app/assets/stylesheets/new_design/procedure_show.scss

```
@@ -45,4 +45,8 @@
     margin-right: 6px;
     display: inline-block;
   }
+
+  p.explication-onglet {
+    margin-bottom: 3 * $default-spacer;
```

Tu penses quoi de centrer les explications au milieu de la page ?

```suggestion
    margin-bottom: 3 * $default-spacer;
    text-align: center;
```